Set Fulu fork epochs for Holesky, Hoodi, and Sepolia testnets

terencechain opened the pull request:
This PR updates the testnet Fulu fork epoch and BPO schedules for Hoodi, Holesky, and Sepolia. Merge only after the corresponding upstream configuration PRs have been merged:

* **Hoodi:** [https://github.com/eth-clients/hoodi/pull/21](https://github.com/eth-clients/hoodi/pull/21)
* **Holesky:** [https://github.com/eth-clients/holesky/pull/132](https://github.com/eth-clients/holesky/pull/132)
* **Sepolia:** [https://github.com/eth-clients/sepolia/pull/111](https://github.com/eth-clients/sepolia/pull/111)

terencechain committed:
Set Fulu fork epochs for Holesky, Hoodi, and Sepolia testnets

nalepae commented:
For Sepolia and Hoodi, do you want to set the ejection balance in an other PR?
(The title of this PR indicates so, but better to double check.)

terencechain commented:
> For Sepolia and Hoodi, do you want to set the ejection balance in an other PR? (The title of this PR indicates so, but better to double check.)

that's a good point, im tempted to do it in the same PR and rename the PR title and description. What do you think?

nalepae commented:
> that's a good point, im tempted to do it in the same PR and rename the PR title and description. What do you think?

That's a good idea IMO, thanks!

prestonvanloon commented:
I think we can pin these WORKSPACE deps to the PR commit, if the upstream PRs are not going to merge today.

terencechain committed:
Update commits

prestonvanloon committed:
Add fulu.yaml to the presets file path loader

prestonvanloon committed:
Add the following placeholder fields:
- CELLS_PER_EXT_BLOB
- FIELD_ELEMENTS_PER_CELL
- FIELD_ELEMENTS_PER_EXT_BLOB
- KZG_COMMITMENTS_INCLUSION_PROOF_DEPTH

prestonvanloon reviewed (approved):
LGTM - Added a check for fulu.yaml presets and excluded the fields that are configured outside of the beaconConfig struct
